main/css/base.css

```diff
@@ -5304,3 +5304,146 @@ i.size-32.icon-new-work{
 {
     height: 52px;
```

Comment:
To ignore permissions changes as in file "archive/.htaccess"

You should use:

git config core.fileMode false

http://stackoverflow.com/questions/1580596/how-do-i-make-git-ignore-file-mode-chmod-changes